manager/manager.go

```diff
@@ -261,6 +303,58 @@ func (m *Manager) issue(volumeID string) error {
 	return nil
 }
 
+func (m *Manager) cleanupStaleRequests(ctx context.Context, log logr.Logger, volumeID string) error {
+	sel, err := m.labelSelectorForVolume(volumeID)
+	if err != nil {
+		return fmt.Errorf("internal error building label selector - this is a bug, please open an issue: %w", err)
+	}
+
+	reqs, err := m.lister.List(sel)
```

Comment:
Quite note: we do a query across _all_ namespaces here, because there's no way for us to know what namespace resources are being created in until `generateRequest` is called (and some weird implementations could do something wacky like create it in a different namespace each time).

I don't think this is much of an issue though, given we match on the `volumeID` and `nodeID` meaning there can't really be a conflict/problem with this approach.

The only oddity I _can_ think of, is if someone were to compromise a single node, they could craft requests to the kubelet to trigger the driver to delete CSRs for some other volume (though it'd still have to be on the same node, as the node ID is also part of the hash).

So all in all, I don't think this represents any kind of additional attack vector 😄